Reply/app/src/main/res/values/styles.xml

```diff
@@ -35,4 +35,15 @@
         <item name="chipStartPadding">0dp</item>
         <item name="android:checkable">false</item>
     </style>
+
+    <style name="ThemeOverlay.Reply.ElevationOverlaysDisabled" parent="">
+        <item name="elevationOverlayEnabled">false</item>
+    </style>
+
+    <style name="Widget.Reply.Tonal.Button" parent="Widget.Material3.Button.UnelevatedButton">
+        <item name="android:backgroundTint">?attr/colorOnSurfaceInverse</item>
+        <item name="android:textColor">?attr/colorOnSurfaceVariant</item>
+        <item name="elevation">@dimen/m3_sys_elevation_level0</item>
```

Comment:
I believe the `UnelevatedButton` style does this for you and you shouldn't need this line